frontend/src/app/Settings/Facility/facilitySchema.ts

```diff
@@ -94,8 +94,11 @@ export const facilitySchema: yup.SchemaOf<RequiredFacilityFields> = yup.object({
     .test(
       "facility-clia",
       ({ value }) => {
-        if (value[2] === "Z" && value.length === 10) {
-          return "Special Z CLIAs are only valid in WA and WY";
+        if (
+          (value[2] === "Z" && value.length === 10) ||
```

Comment:
Can the length ever be shorter than 3 here?  Might be better to check the length first.